src/Messages.js

```diff
@@ -466,6 +466,11 @@ export default defineMessages({
         description: 'A link label to access.redhat.com ',
         defaultMessage: 'View packages and errata at access.redhat.com'
     },
+    noAdvisoryAvailable: {
```

Comment:
We had an agreement. It would be better if we follow it: we agreed that we will not use internationalization (intl) for new features/wordings. What do you think?

Reply:
must have missed that conversation, removed the call to `intl`